crates/turbopack-ecmascript-runtime/js/src/dev/runtime/base/runtime-base.ts

```diff
@@ -332,6 +332,7 @@ function instantiateModule(id: ModuleId, source: SourceInfo): Module {
           m: module,
           c: moduleCache,
           l: loadChunk.bind(null, { type: SourceType.Parent, parentId: id }),
+          w: loadWebAssembly.bind(null, { type: SourceType.Parent, parentId: id }),
```

Comment:
```
const sourceInfo = { type: SourceType.Parent, parentId: id };
```

```suggestion
          l: loadChunk.bind(null, sourceInfo),
          w: loadWebAssembly.bind(null, sourceInfo),
```

let's be a bit more efficient since this is called a lot